smithy-openapi/src/main/java/software/amazon/smithy/openapi/OpenApiVersion.java

```diff
@@ -46,6 +53,11 @@ public boolean supportsContentEncodingKeyword() {
         return supportsContentEncodingKeyword;
     }
 
+    @SmithyInternalApi
+    public Set<String> getUnsupportedKeywords() {
```

Comment:
Oh interesting to make this version specific. Is this internal because you aren't sure about the design? Maybe unstable instead if that's the concern?